examples/concurrent/main.go

```diff
@@ -40,16 +40,19 @@ func run(contextDir, dockerfile, imageName, remoteBuildURI, volumeSource, volume
 		RemoteURI:  remoteBuildURI,
 		StdIN:      input,
 	}
+	containerOpts := &provision.ContainerOptions{}
+	if volumeSource != "" {
+		if volumeDestination == "" {
+			volumeDestination = volumeSource
```

Comment:
if I do `if  volumeSource != ""  && volumeDestination == ""`, I always set destination equals source.
The first verify if we have volumes, the second verify if destination is empty, are different conditions.